frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,46 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<array{name: string}>
```

Comment:
De acuerdo al cambio, esto puede ser así:
```suggestion
     * @return list<array{alias: string}>
```